susemanager-utils/testing/automation/backend-unittest-pgsql.sh

```diff
@@ -1,7 +1,25 @@
 #! /bin/sh
+if [ -z ${PRODUCT+x} ];then
+    PRODUCT="Uyuni"
+fi
+
+while getopts 'P:h' c
+do
+    case $c in
+        P) PRODUCT=$OPTARG ;;
+        h) echo "Usage $0 [-p PRODUCT]";exit -2;;
+    esac
+done
 
 HERE=`dirname $0`
-. $HERE/VERSION
+
+if [ ! -f $HERE/VERSION.${PRODUCT} ];then
+   echo "VERSION.${PRODUCT} does not exist"
+   exit -3
+fi
+
+echo "Loading VERSION.${PRODUCT}"
+. $HERE/VERSION.${PRODUCT}
 GITROOT=`readlink -f $HERE/../../../`
```

Comment:
```suggestion
GITROOT=`readlink -f ${HERE}/../../../`
```
You are not changing it here, but you are changing it at the other files, so we can as well fix it everywhere :-)